docs/tutorials/best-practices/error-safety.md

```diff
@@ -166,7 +166,7 @@ what should be done about it.
 
 ```Lua hl_lines="4-7 24"
 local number = scope:Value(1)
-local double = scope:Computed(function()
+local double = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
local double = scope:Computed(function(use)
```

Reply:
removed